src/API/Startup.cs

```diff
@@ -17,18 +20,67 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            services
+                .AddMvcCore()
+                .AddVersionedApiExplorer(o => o.GroupNameFormat = "'v'VV");
+
             services.AddMvc();
```

Comment:
yep, it isn't necessary. I will remove it. It escaped.

Reply:
This is needed, otherwise API won't work. Since AddMvc adds a lot of components that aren't designated as core.

Example here: https://offering.solutions/blog/articles/2017/02/07/difference-between-addmvc-addmvcore/

AddMvc() returns an IMvcBuilder while AddMvcCore return a IMvcCoreBuilder
We either add all dependencies or maintain this like this.

Reply:
here's another link for this specific case of api versioning: https://github.com/Microsoft/aspnet-api-versioning/wiki/API-Documentation#aspnet-core